src/Command/Run.php

```diff
@@ -55,7 +55,8 @@ protected function execute(
     ): int {
         try {
             // Print intro
-            $output->writeln(\sprintf('<fg=yellow;options=bold>%s</> <info>v%s</>', Info::NAME, Info::VERSION));
+            $output->writeln(\sprintf('<fg=yellow;options=bold>%s</> <info>v%s</>:', Info::NAME, Info::VERSION));
+            trap(Info::JOKES[\array_rand(Info::JOKES)]);
```

Comment:
```suggestion
            $output->writeln(\sprintf('<fg=yellow;options=bold>%s</> <info>v%s</>', Info::NAME, Info::VERSION));
```

I think we don't really need to print jokes every run

Reply:
deleted